app/views/reports/regions/_header.html.erb

```diff
@@ -49,7 +49,7 @@
           %>
             <button class="dropdown-item query-filter-month <%= 'active' if selected %>" name="period[value]"
                     type="submit" data-period-type="month" value="<%= period.value %>">
-              <%= period.value.to_s(:month_year) %>
+              <%= period.value.to_s(:mon_year) %>
```

Comment:
I'm unsure if the `Period` is smart enough to just invoke `to_s` on it directly. It might be (it already is for months, but quarters are a bit more involved). TBH this diff is from a date format rename, so I'll defer cleaning this part up to the next reports PR.